.github/workflows/wheels.yml

```diff
@@ -30,7 +30,80 @@ env:
   FORCE_COLOR: 1
 
 jobs:
-  build:
+  build-1-QEMU-emulated-wheels:
+    name: QEMU ${{ matrix.python-version }} ${{ matrix.spec }}
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        python-version:
+          - pp39
+          - pp310
+          - cp38
+          - cp39
+          - cp310
+          - cp311
+          - cp312
+        spec:
+          - manylinux2014
+          - manylinux_2_28
+          - musllinux
+        exclude:
+          - { python-version: pp39, spec: musllinux }
+          - { python-version: pp310, spec: musllinux }
+
+    steps:
+      - uses: actions/checkout@v4
+        with:
+          submodules: true
+
+      - uses: actions/setup-python@v5
+        with:
+          python-version: "3.x"
+
+      # https://github.com/docker/setup-qemu-action
+      - name: Set up QEMU
+        uses: docker/setup-qemu-action@v3
+
+      # https://github.com/docker/setup-buildx-action
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v3
```

Comment:
Maybe this is obvious and I'm missing it, but why do we need this step?

Reply:
Because I copied and pasted from another project where it's probably also not needed :)

https://cibuildwheel.readthedocs.io/en/stable/faq/#emulation shows only `docker/setup-qemu-action` and `docker/setup-buildx-action`

I'll check if we can remove it.

Reply:
Yep, not needed 👍

Reply:
Thanks